src/pages/_document.tsx

```
@@ -29,6 +29,17 @@ export default function Document() {
         <meta property="og:type" content="website" />
         <meta property="og:image" content={`${APP_URL}/logo.png`} />
         <meta property="og:description" content={APP_DESCRIPTION} />
+
+        <script
+          defer
+          data-domain="usenexus.org"
+          src="https://plausible.io/js/script.outbound-links.js"
+        ></script>
+        <script
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)